### ISSUE TITLE
Play-ws (scamandrill and scamandrill_2.4 3.0.0)release roadmap

### ISSUE THREAD

**Issue by graingert** [scamandrill/scamandrill#29]
* [ ] add documentation for the new Play-ws version
* [ ] add transition documentation
* [ ] release to Bintray, get some test results from people using Play 2.4 and Play 2.5 and no Play.
* [ ] release to Maven

**Comment by graingert** [comment]
@joelsonoda please tick these if there are any that are already done, or add more as appropriate.

**Comment by joelsonoda** [comment]
I'll add a migration section to the gh-pages branch along with the documentation on using play 2.4 and play 2.5

**Comment by joelsonoda** [comment]
I've added the migration documentation to the 2-5-release-docs

**Comment by joelsonoda** [comment]
@graingert should be able to release to bintray by using `sbt release` on master as well as on 2.4.x and filling in the appropriate version numbers (I imagine we could update them to 2.5.1-SNAPSHOT and 2.4.1-SNAPSHOT respectively).

Maybe @gmalouf would be able to help with the non-play testing once we've released to bintray.

**Comment by joelsonoda** [comment]
@graingert You should be able to publish the two versions to bintray using the following:
`
git checkout master & sbt 'release with-defaults'
`
and
`
git checkout 2.4.x & sbt 'release with-defaults'
`

**Comment by joelsonoda** [comment]
@graingert once this is done, I'll update my projects here to use the new versions and proxy bintray through our local maven repo. Any outstanding issues?

**Comment by gmalouf** [comment]
is this project still ongoing?

**Comment by joelsonoda** [comment]
@gmalouf, we use it extensively at our company (2.6.0 deployed to a private maven repo). There are branches available for play 2.4/2.5/2.6 and the version currently released to maven central will work fine if you aren't integrating with one of the newer versions of play (though some of the models have been re-worked to more accurately reflect the mandrill api).
